warehouse/router/tracker.go

```diff
@@ -23,7 +23,14 @@ import (
 // CronTracker Track the status of the staging file whether it has reached the terminal state or not for every warehouse
 // we pick the staging file which is oldest within the range NOW() - 2 * syncFrequency and NOW() - 3 * syncFrequency
 func (r *Router) CronTracker(ctx context.Context) error {
+	tick := r.statsFactory.NewTaggedStat("warehouse_cron_tracker_tick", stats.GaugeType, stats.Tags{
+		"module":   moduleName,
+		"destType": r.destType,
+	})
 	for {
+
+		tick.Count(1)
```

Comment:
Shouldn't this be guage?
```suggestion
		tick.Gauge(1)
```

Reply:
Just updated the description.

The idea is to be a count, so we can detect when the counter is not increasing.